README.md

```diff
@@ -26,7 +26,7 @@ Commands:
   apply   check and apply changes
   check   check syntax/config
   import
-  login   store login config per url
+  login   store plaintext creds per url
```

Comment:
I don't think "plaintext creds" is clearer than "login config". It's an implementation detail.

Reply:
How about "plaintext login config"?

Reply:
Dropping the plaintext aspect since pretty much every other jenkins tool has this affliction.